lib/next_ls/helpers/ast_helpers/env.ex

```diff
@@ -113,6 +113,8 @@ defmodule NextLS.ASTHelpers.Env do
     }
   end
 
+  def ascend(nil, acc, _callback), do: acc
+
```

Comment:
is this change necessary?

Reply:
Yes

without it
```elixir
defmod|
```
don't trigger autocomplete because there is no previous node 

line here: https://github.com/elixir-tools/next-ls/pull/398/files/9d65b759ac0cf86c943190342cf829097fc62346#diff-934a3387a31dbe6cdfbab7475b2f2c24722ff71a7c11b282ef363ac6952f1c9fR19